src/rgw/rgw_rest_group.h

```diff
@@ -0,0 +1,105 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+#ifndef CEPH_RGW_REST_GROUP_H
+#define CEPH_RGW_REST_GROUP_H
+
+class RGWRestGroup : public RGWRESTOp {
+protected:
+  string group_name;
```

Comment:
We don't have `using namespace std` in headers any more, so this should be std::